app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+  validates :email, presence: true,
+            format:{with:/\S+@\S+/},
+            uniqueness: {case_sensitive: false}
+  validates :password, length: { minimum: 5, allow_blank: true }
+  has_secure_password
```

Comment:
I think devise applies this model validation so there’s no need to double check. Also, I think if you double validate an attribute the error message will appear twice